docker-app/worker_wrapper/wrapper.py

```diff
@@ -323,46 +323,44 @@ def before_docker_run(self) -> None:
         self.data_last_packaged_at = timezone.now()
 
     def after_docker_run(self) -> None:
-        # only successfully finished packaging jobs should update the Project.data_last_packaged_at
-        if self.job.status == Job.Status.FINISHED:
```

Comment:
This check was never necessary, as before the `job.status = Job.status.FINISHED` was set right before calling `after_docker_run` .